doc/ref_cert/lfn/chapters/chapter02.md

```diff
@@ -14,38 +14,58 @@
 
 <a name="2.1"></a>
 ## 2.1 Introduction
-
-Describe the purpose.. 
-
-- Define E2E Framework Requirements, process and automation for NFVI Certification
-- Ensure Platform Stability, Operability, & Performance for target RAs
+NFVI (Network Functions Virtualization Infrastructure) refers to the physical resources (compute, storage and network) on which virtual network functions (VNFs) are deployed. As such, the performance of a VNF depends on the underlying NFVI over which it is hosted. A certain VNF may perform good in one hardware and may perform worst in another. Thus, a need arises to certify NFVI that can help in onboarding VNFs onto a hardware with an acceptable level of VNF performance. Certain frameworks like [yardstick](https://github.com/opnfv/yardstick), [vsperf](https://github.com/opnfv/vswitchperf) etc. provide a set of tests that can be run on a hardware to obtain Key Performance Indicators (KPIs) which give a measurable output of the NFVI's performance. With these KPIs, a decision can be made on the VNFs that can offer an acceptable level of performance when on-boarded on the NFVI.
 
 <a name="2.2"></a>
 ## 2.2 Methodology
-
-Define the approach to NFVI testing, and Frameworks needed...
+The NFVI provided by hardware vendors is consumed or used by VNFs via APIs exposed by Virtualised Infrastructure Manager (VIM). The resources created by VIM on the NFVI use the underlying physical hardware (compute, storage and network) either directly or indirectly. CNTT recommends RA1/RA2 to be used as a reference architecture for NFVI certification. This  would provide a set of standard interaces to create resources on NFVI. Below step by step process illustrates the NFVI certification methodology:
+* CNTT RA1/RA2 is deployed on NFVI that needs to be certified.
+* A set of tests are run on NFVI to determine the NFVI readiness for certification process,
+* Golden KPIs are taken as a reference.
+* A set of tests are run on the target NFVI.
+* KPIs obtained from the target NFVI are collected and submitted to certification portal.
+* The NFVI KPIs are reviewed and compared with Golden KPIs to determine if the certification badge is to be provided to NFVI or not.
 
```

Comment:
Add "Verification Methodologies to be implemented, from a process perspective include:"

1. Engineering package validations will be performed against targeted infrastructure/architecture.
2. Configuration settings/features/capabilities will be baseline
3. Entrance Criteria Guidelines will be satisfied prior to OPNFV verification (i.e. Supplier needs to submit/agree/conform)
    4. Conform to CNTT ref model
    5. Confirm to ref arch
    6. Submit standard documentation
    7. Adhere to security compliance
8. Exit Criteria Guidelines will be satisfied prior to issuance of OPNFV compliance badges.
    9. Certification for Tenants (OVP Ecosystem)
    10. Operaitonal Considerations (Telco Perspective)
    11. End User Considerations (Telco Perspective)
12. Test runs using reference model VNFs will validate RA chosen by the VNF-supplier meets developer needs
13. Verification decisions will be based on data. Test harness is “compatible”, or “conforms” to testing against standard interfaces and services
14. VNF functionality easily tested w/ addition of Supplier Apparatus
15. Leverage test harnesses from existing open source projects where practical, and applicable.

Reply:
Added

Reply:
Let me know if we can resolve this.

Reply:
Looks good.  Closing/Resolving now.